lib/rules/web/admin_console/4.4/monitoring_alerts.xyaml

```diff
@@ -0,0 +1,112 @@
+click_icon_index_button:
+  element:
+    selector:
+      xpath: (//button[@aria-label='<aria_label_text>'])[<index>]
+    op: click
+set_placeholder_input_value:
+  element:
+    selector:
+      xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
+    op: send_keys <input_value>
+    type: input
+clear_label_input_value:
```

Comment:
`clear_label_input_value` will not be needed since `set_label_input` already cleared the input first

Reply:
move to set_label_input_simple: